plugin/chatgpt.vim

```diff
@@ -257,6 +260,7 @@ vnoremap <silent> <Plug>(chatgpt-menu) :call ChatGPTMenu()<CR>
 command! -range -nargs=? Ask call SendHighlightedCodeToChatGPT('Ask',<q-args>)
 command! -range -nargs=? Explain call SendHighlightedCodeToChatGPT('explain', <q-args>)
 command! -range Review call SendHighlightedCodeToChatGPT('review', '')
+command! -range Document call SendHighlightedCodeToChatGPT('document', '')
```

Comment:
Last comment than this is good to go and i'll merge.  ```suggestion
command! -range -nargs=? Document call SendHighlightedCodeToChatGPT('document', <q-args>)
```

This will allow the command to pass additional context to the prompt.